docs/api-development/Asynchronous APIs/04-Protocols.md

```diff
@@ -29,6 +29,11 @@ HTTP has limited scalability for handling a large number of connections and is n
 ### Proprietary protocols
 
 There are a number of message broker technologies which implement their own protocols such as [Solace Message Format (SMF)](https://docs.solace.com/Messaging/SMF-Topics.htm) or the [Kafka protocol](https://kafka.apache.org/0100/protocol.html). These proprietary protocols **MAY** be used, but **MUST NOT** be the only protocol offered by the API provider. This is to ensure that API Consumers or Message Producers who may have existing implementations using open protocols do not need to undertake more work to integrate, or become reliant on a particular broker technology.
+
+## APIs
+
+### JMS (Java Message Service)
```

Comment:
Have extended the scope of this page slightly as JMS is not a protocol as with the others listed on this page, but is worth calling out specifically due it's wide usage